connector_importer_product/components/product_product/record_handler.py

```diff
@@ -184,8 +185,13 @@ def _update_template_attributes(self, odoo_record, values, orig_values):
         else:
             odoo_record.product_template_attribute_value_ids = valid_tpl_attr_values
 
+    def _find_attr(self, attr_column, orig_values):
+        """Find matching attribute."""
+        attr_xid = sanitize_external_id(attr_column)
+        return self.env.ref(attr_xid)
```

Comment:
Shouldn't we use raise_if_not_found here?

Reply:
no